### DISCUSSION TITLE
How do I change (or remove) the path on the target host

### DISCUSSION THREAD

**Discussion by edhCoder** [apache/apisix#11455] [General]
I want to create a route that accepts a pathname at the end of the uri: 
    http://myserver.com:9080/some_path
and on the upstream host I don't want anything after the hostname, resolving to http://desthost.com/
Is that possible with APISIX? If so, what is the syntax on the input? 

Desired action: 
input:  http://myserver.com:9080/go   dest: http://google.com/

**Comment by kayx23** [comment]
Check out proxy-rewrite plugin.

**Comment by GongchuangSu** [comment]
This can be done using the proxy-rewrite plugin, e.g. the regex_uri attribute is set to ["^.*", "/"]
